rules/windows/file_rename/file_rename_win_not_dll_to_dll.yml

```diff
@@ -7,18 +7,18 @@ references:
     - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1036/T1036.md#atomic-test-1---system-file-copied-to-unusual-location
 author: frack113
 date: 2022/02/19
-modified: 2022/03/13
+modified: 2022/05/13
 logsource:
     product: windows
     category: file_rename
 detection:
     to_dll:
         TargetFilename|endswith: '.dll'
     filter_from_dll:
-        - OriginalFilename|endswith: 
+        - OriginalFilename|endswith:
             - '.dll'
             - '.tmp'  # VSCode FP
-        - OriginalFilename|contains: 
+        - OriginalFilename:
```

Comment:
This is a `file_rename` from ETW kernel-file-rename not a `process_creation`.
You can see `OriginalFilename` not `OriginalFileName`
`N` make the diff 😄